tests/extras/test_translator.py

```diff
@@ -25,23 +25,31 @@
 }
 
 
+@pytest.mark.parametrize("tlr_fixture_name", ("rest_tlr","tlr"))
```

Comment:
I assume you tried to pass `rest_tlr` and `tlr` (unquoted) in the parametrize setup, right? I can see what's happening, but it's unclear to me why this indirection is necessary. (It's fundamentally the same as passing functions to any other function, I think.)

So, please just confirm that the lookup/indirection is needed. If it's needed, then fine (and nice workaround!).

Reply:
Correct. Quotes are necessary due to [request.getfixturevalue](https://docs.pytest.org/en/6.2.x/reference.html#pytest.FixtureRequest.getfixturevalue), whose argument is a string, on the following lines.